CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Samuel Giddins](https://github.com/segiddins)
   [cocoapods-try#31](https://github.com/CocoaPods/cocoapods-try/issues/31)
 
+* Supports running pre-install hooks in plugins. This happens before the resolver
+  does its work, and offers easy access to the sandbox, podfile and lockfile via a 
+  PreInstallHooksContext object. This also renames the post-install hooks from HooksContext
+  to PostInstallHooksContext.
```

Comment:
Ouch, do we have to tell everyone who make a changelog change that? Should probably get linted in `rake spec` - I'd have never thought of doing that.